jajajaden01/taskCreation
# Adding a task creation on backend and frontend

#### What does this PR do?
This PR adds a task creation on backend and frontend
#### Description of Task to be completed?
I've backend and frontend
#### How should this be manually tested?
- clone the repo
on Backend:
- go inside /backend by typing `cd backend` in terminal
- make sure you have a postgres DB running, and share the credentials in `.env` file referring on `.env.example`
- then run `npm install` to install the packages after that run `npm run dev`
on Frontend:
- go inside /frontend by typing `cd frontend` in terminal
- then run `npm install` to install the packages after that run `npm run start`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories? link on redmine issues platform
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
